salt/defaults/exitcodes.py

```diff
@@ -34,5 +34,8 @@
 
 # The Salt specific exit codes are defined below:
 
+# keepalive exit code is a hint that the process should be restarted
+SALT_KEEPALIVE = 99
```

Comment:
This config is the keepalive interval? Or the exit code? Probably should rename the option to make it more clear:)